pkg/gitops/flux/installer.go

```diff
@@ -133,6 +135,28 @@ func NewInstaller(ctx context.Context, k8sRestConfig *rest.Config, k8sClientSet
 
 // Run runs the Flux installer
 func (fi *Installer) Run(ctx context.Context) error {
+	fluxNSExists, err := kubernetes.CheckNamespaceExists(fi.k8sClientSet, fi.opts.Namespace)
+	if err != nil {
+		return errors.Wrapf(err, "cannot check if namespace %s exists", fi.opts.Namespace)
+	}
+	if fluxNSExists {
+		logger.Info("Namespace \"%s\" already exists. eksctl will now delete all its content before proceeding to install Flux.", fi.opts.Namespace)
```

Comment:
Our current style convention for log messages dictates that they always start with a lower-case character (unless a name is used).

Reply:
Noted, will fix. Thanks 👍